URBNSwiftCarousel/Classes/URBNSwCarouselTransitionController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  URBNSwCarouselTransitionController.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 4/25/16.
+//
+//
+
+
+/**
+ Animation controller built to handle transitions between two images in separate view controllers.
+ By default, this class supports non-interactive transitions. To enable interactive transitions, register
+ a view for them using the registration method provided.
+ 
+ This class implements the UIViewControllerTransitioningDelegate protocol. As a consumer, you only need to implement
+ the URBNCarouselTransitioning protocol.
+ 
+ For the time being, this controller only supports transitions between images.
+ */
+
+
+public class URBNSwCarouselTransitionController: NSObject, UIViewControllerAnimatedTransitioning, UIViewControllerTransitioningDelegate {
+    private weak var context: UIViewControllerContextTransitioning?
+    private var transitionView = UIImageView()
+    private var originalSelectedCellFrame = CGRectZero
+    private var springCompletionSpeed: CGFloat = 0.7
+    private var sourceViewController: UIViewController?
+    public var presentationController: URBNPresentationController?
+    
+    // MARK: UIViewControllerAnimatedTransitioning Delegate Methods - Non-Interactive
+    public func transitionDuration(transitionContext: UIViewControllerContextTransitioning?) -> NSTimeInterval {
+        return 0.5
+    }
+    
+    public func animateTransition(transitionContext: UIViewControllerContextTransitioning) {
+        
+        guard let fromVC = transitionContext.viewControllerForKey(UITransitionContextFromViewControllerKey),
```

Comment:
Might want to do some asserts or something for these guards.  Otherwise the implementer will be left wandering  down the rabbit hole trying to figure out why it's not working